src/targets/node/fetch/client.ts

```diff
@@ -137,19 +131,16 @@ export const fetch: Client = {
     blank();
 
     if (includeFS) {
-      unshift("const fs = require('fs');");
-    }
-    if (postData.params && postData.mimeType === 'multipart/form-data') {
-      push('options.body = formData;');
-      blank();
+      unshift("import fs from 'fs';\n");
     }
+
     push('fetch(url, options)');
     push('.then(res => res.json())', 1);
     push('.then(json => console.log(json))', 1);
-    push(".catch(err => console.error('error:' + err));", 1);
+    push('.catch(err => console.error(err));', 1);
 
     return join()
       .replace(/'encodedParams'/, 'encodedParams')
-      .replace(/"fs\.createReadStream\(\\"(.+)\\"\)"/, 'fs.createReadStream("$1")');
+      .replace(/'formData'/, 'formData');
```

Comment:
because we're constructing the code snippet as `reqOps.body = formData` and then stringifying that into the snippet, `formData` gets put into the snippet as `'formData'`. this undoes that to make `formData` in the snippet be treated as the variable we're constructing instead.

Reply:
ah missed those quotation marks. makes sense